src/frontend/components/StudentLiveAdvertising/StudentLiveRegistration/index.spec.tsx

```diff
@@ -226,10 +260,13 @@ describe('<StudentLiveRegistration />', () => {
   });
 
   it('does not render if we fail to initialize current subscription for lti user', async () => {
-    mockGetLiveSessions.mockRejectedValue(undefined);
+    const deferred = new Deferred();
+    fetchMock.get('/api/livesessions/?limit=999', deferred.promise);
 
     const { elementContainer: container } = render(<StudentLiveRegistration />);
 
+    deferred.resolve(400);
+
     expect(container!.childNodes.length).toEqual(0);
   });
```

Comment:
We both know this test does not cover anything (since it passes with a falty implementation).
I have no idea yet how to fix this tho

Reply:
I removed it.